### PR TITLE
Updated screenshots

### PR DESCRIPTION
**Pull request by davidbenque**

### CODE REVIEW

File: images/app-creation-screen.png
Comment (davidbenque) [inline images/app-creation-screen.png:1]: element needs to be captured instead of page

File: images/app-richtext-preview.png
Comment (davidbenque) [inline images/app-richtext-preview.png:1]: add comments and outline to the demo document to show these features?

File: images/calendar.png
Comment (davidbenque) [inline images/calendar.png:1]: calendar view does not show much anymore. Could need a dummy caldav file to fill up using either month or week view

File: images/drive-history.png
Comment (davidbenque) [inline images/drive-history.png:1]: history build up on the demo drive

File: images/history-toolbar-sheets.png
Comment (davidbenque) [inline images/history-toolbar-sheets.png:1]: build up some history in the demo sheet?

File: images/snapshots-dialog.png
Comment (davidbenque) [inline images/snapshots-dialog.png:1]: UI details are missing but not a blocker

File: images/drive-history.png
Comment (JanRei) [inline images/drive-history.png:1]: Great to see that the screenshots are being updated. Regarding this particular file, I wonder if it is actually used in the documentation. There is no screenshot in the section about [drive history](https://docs.cryptpad.org/en/user_guide/drive.html#history).

Comment (davidbenque) [reply images/drive-history.png:1]: indeed! thank you for noticing

**Pull request merged**